pypeln/process/queue.py

```diff
@@ -18,13 +23,13 @@
 
 class IterableQueue(Queue, tp.Generic[T], tp.Iterable[T]):
```

Comment:
this inheritance of Queue might complicate things

Reply:
Typehints should stay the same, the important thing would be to use a different module when initializing `Queue` and `Process` at runtime.

Reply:
heres a wip impl https://github.com/cgarciae/pypeln/pull/100